From jmm@debian.org  Wed Jan 21 16:32:30 2009
From: Moritz Muehlenhoff <jmm@debian.org>
Date: Wed, 21 Jan 2009 22:00:43 +0100
Subject: Staging: wlan-ng: Cleanup wlan_compat.h more
To: greg@kroah.com
Cc: Moritz Muehlenhoff <jmm@debian.org>
Message-ID: <1232571646-32579-4-git-send-email-jmm@debian.org>


Move version identifier into p80211netdev.h and rename it to 0.3.0-staging
to differentiate from the out-of-tree version.

Also remove the unused wlan_ethconv declaration.

Signed-off-by: Moritz Muehlenhoff <jmm@debian.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/wlan-ng/p80211netdev.h |    2 ++
 drivers/staging/wlan-ng/wlan_compat.h  |    7 -------
 2 files changed, 2 insertions(+), 7 deletions(-)

--- a/drivers/staging/wlan-ng/p80211netdev.h
+++ b/drivers/staging/wlan-ng/p80211netdev.h
@@ -59,6 +59,8 @@
 /*================================================================*/
 /* Constants */
 
+#define WLAN_RELEASE	"0.3.0-staging"
+
 #define WLAN_DEVICE_CLOSED	0
 #define WLAN_DEVICE_OPEN	1
 
--- a/drivers/staging/wlan-ng/wlan_compat.h
+++ b/drivers/staging/wlan-ng/wlan_compat.h
@@ -93,8 +93,6 @@
 #define WLAN_DBVAR	wlan_debug
 #endif
 
-#define WLAN_RELEASE	"0.3.0-lkml"
-
 #include <linux/hardirq.h>
 
 #define WLAN_LOG_ERROR(x,args...) printk(KERN_ERR "%s: " x , __func__ , ##args);
@@ -179,10 +177,5 @@ typedef struct net_device netdevice_t;
 extern int wlan_debug;
 #endif
 
-extern int wlan_ethconv;		/* What's the default ethconv? */
-
-/*=============================================================*/
-/*--- Functions -----------------------------------------------*/
-/*=============================================================*/
 #endif /* _WLAN_COMPAT_H */
 
